docs/src/main/sphinx/installation/containers.md

```diff
@@ -1,9 +1,17 @@
 # Trino in a Docker container
 
-The Trino project provides the [trinodb/trino](https://hub.docker.com/r/trinodb/trino)
-Docker image that includes the Trino server and a default configuration. The
-Docker image is published to Docker Hub and can be used with the Docker runtime,
-among several others.
+The Trino project provides the default
+[trinodb/trino](https://hub.docker.com/r/trinodb/trino) Docker image that
+includes the Trino server, all plugins, and a default configuration.
+
+The additional [trinodb/trino-core](https://hub.docker.com/r/trinodb/trino-core)
+Docker images contains only a minimal set of essential plugins. The
```

Comment:
> a minimal set of essential plugins

Could you clarify which plugins are included? From the current sentence, it’s unclear when `trino-core` should be used.